repo2docker/buildpacks/conda/environment.yml

```
@@ -4,5 +4,6 @@ dependencies:
   - jupyterlab==1.2.6
   - jupyterhub-singleuser==1.1.0
   - nbconvert==5.6.1
+  - nodejs==13.8
```

```suggestion
- nodejs==12.*
```

Let's use only major-version pinning on nodejs and use LTS (always even) rather than latest.